LoopKitUI/Views/ActivityIndicator.swift

```diff
@@ -12,14 +12,18 @@ public struct ActivityIndicator: UIViewRepresentable {
 
     @Binding var isAnimating: Bool
     let style: UIActivityIndicatorView.Style
+    let color: UIColor?
     
-    public init(isAnimating: Binding<Bool>, style: UIActivityIndicatorView.Style) {
+    public init(isAnimating: Binding<Bool>, style: UIActivityIndicatorView.Style, color: UIColor? = nil) {
```

Comment:
[nit] should this default to `nil` or the default activity indicator color?

Reply:
The `style` property implicitly sets `color`.